### ISSUE TITLE
Bug in Regulator

### ISSUE THREAD

**Issue by sindunuragarp** [felipemoraes/searchx-backend#27]
When searching using the bing provider, I get this error.

```
16:56:43 worker.1   |  Starting Workers
16:56:44 api.1      |  Starting Server
16:56:44 api.1      |  SearchX API is running on port 4443
16:57:01 api.1      |  _anonymous_ has connected (/session#8LeACNGEiCSiWI8SAAAB)
16:57:07 api.1      |  _anonymous_ has connected (/session#pBPNY8RUEjHsRvl5AAAC)
16:57:15 api.1      |  TypeError: Cannot read property 'replace' of undefined
16:57:15 api.1      |      at results.map.result (/home/code/searchx-backend/app/services/search/regulator.js:92:39)
16:57:15 api.1      |      at Array.map (<anonymous>)
16:57:15 api.1      |      at addMissingFields (/home/code/searchx-backend/app/services/search/regulator.js:88:20)
16:57:15 api.1      |      at Object.exports.fetch (/home/code/searchx-backend/app/services/search/regulator.js:73:26)
16:57:15 api.1      |      at <anonymous>
16:57:15 api.1      |      at process._tickCallback (internal/process/next_tick.js:118:7)
```

**Comment by sindunuragarp** [comment]
Also it seems that the Indri provider requires a dependency that is not provided inside the repository

```
17:00:22 worker.1   |  Starting Workers
17:00:23 api.1      |  module.js:545
17:00:23 api.1      |      throw err;
17:00:23 api.1      |      ^
17:00:23 api.1      |  Error: Cannot find module '../../../../lib/node-indri/node-indri'
17:00:23 api.1      |      at Function.Module._resolveFilename (module.js:543:15)
17:00:23 api.1      |      at Function.Module._load (module.js:470:25)
17:00:23 api.1      |      at Module.require (module.js:593:17)
17:00:23 api.1      |      at require (internal/module.js:11:18)
17:00:23 api.1      |      at Object.<anonymous> (/home/code/searchx-backend/app/services/search/providers/indri.js:3:15)
17:00:23 api.1      |      at Module._compile (module.js:649:30)
17:00:23 api.1      |      at Object.Module._extensions..js (module.js:660:10)
17:00:23 api.1      |      at Module.load (module.js:561:32)
17:00:23 api.1      |      at tryModuleLoad (module.js:501:12)
17:00:23 api.1      |      at Function.Module._load (module.js:493:3)
17:00:23 api.1      |      at Module.require (module.js:593:17)
17:00:23 api.1      |      at require (internal/module.js:11:18)
17:00:23 api.1      |      at Object.<anonymous> (/home/code/searchx-backend/app/services/search/provider.js:5:15)
17:00:23 api.1      |      at Module._compile (module.js:649:30)
17:00:23 api.1      |      at Object.Module._extensions..js (module.js:660:10)
17:00:23 api.1      |      at Module.load (module.js:561:32)
17:00:23 api.1      |  error: Forever detected script exited with code: 1
17:00:24 api.1      |  error: Script restart attempt #1
```

**Comment by kiliangrashoff** [comment]
fixed by #29
